src/Fulbank/Classes/Database.cs

```diff
@@ -18,34 +20,33 @@ public class Database
         #region Constructor
             public Database()
             {
-                this._db = "fulbank";
-                this._user = "root";
-                this._psswd = "''";
-                this._host = "localhost";
-                this._data = "server=" + this._host + ";database=" + this._db + ";uid=" + this._user + ";password=" + this._psswd + ";SSL MODE='None'";
+                _db = "fulbank";
+                _user = "root";
+                _psswd = "''";
+                _host = "localhost";
+                _data = "server=" + _host + ";database=" + _db + ";uid=" + _user + ";password=" + _psswd + ";SSL MODE='None'";
```

Comment:
Ah finalement tu retire les `this`, par conte la concaténation xD

```suggestion
                _db = "fulbank";
                _user = "root";
                _psswd = "''";
                _host = "localhost";
                _data = $"server={_host};database={_db};uid={_user};password={_psswd};SSL MODE='None'";
```